src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -190,6 +190,10 @@ void primitive_inst::update_shape() {
         }
         auto& dep = _node->get_dependency(i);
         auto dep_id = dep.id();
+        // exclude fused node from memory_deps
+        if (i >= _node->get_primitive()->input_size()) {
```

Comment:
I think line 188 can be modified instead: `_node->get_dependencies().size()` -> `_node->get_primitive()->input_size()`

Reply:
I've changed to use fused_primitive_desc::dep_start_idx instead of get_primitive()->input_size() because get_primitive() doesn't represent original prim.